CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## v2.7.0 / 2022-11-18
+
+Note: Experimental VerticalPodAutoscaler metrics are considered deprecated in this release and will be removed in v2.9.0.
+Please use CustomResourceStateMetrics to gather metrics from VPA resources. See: #1718
+
+Note: Experimental CustomResourceState has been redesigned for better usability. See: #1847
+
+* [CHANGE]      Deprecate VerticalPodAutoscaler metrics #1835 @rexagod
+* [FEATURE]     Add experimental kube_pod_container_status_last_terminated_exitcode metric #1752 @ssabo
+* [FEATURE]     Introduce custom-resources-only flag #1813 @bavarianbidi
+* [FEATURE]     Allow allowlist filtering by wildcard key #1823 @rexagod
+* [FEATURE]     Handle singular labels in allowlist #1826 @rexagod
```

Comment:
- This seems to replicated here: https://github.com/kubernetes/kube-state-metrics/pull/1896/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR21.
- Pasting some PRs below that I don't see here, in case any of them can be included in the changelog.
#1834
#1849
#1878
#1880
#1890

Reply:
We try to only include user visible changes, looks like I missed some of the ones you mentioned! Sorry for that.

Reply:
I've included the missing user visible ones.